parastell/invessel_build.py

```diff
@@ -698,6 +691,59 @@ def logger(self):
     def logger(self, logger_object):
         self._logger = log.check_init(logger_object)
 
+    @property
+    def separate_chamber(self):
+        return self._separate_chamber
+    
+    @separate_chamber.setter
+    def separate_chamber(self, value):
```

Comment:
This method does a lot, but I'm not entirely sure the use case.  It looks like it may be set up to switch between true/false values of `separate_chamber` but that seems like an unlike case to me.  If we don't ever allow that to happen, is all of this still necessary?

Reply:
Yes, much of the architecture here is make the class robust against switching the value when imported into a script. If we assume the user won't try to do so, or put in a protection against it, much of the code can be removed. The same goes for some of the other setters.